src/amp-viewer-host.js

```diff
@@ -22,5 +22,139 @@ import {
 } from 'amp-viewer-messaging/messaging';
 
 
+const CHANNEL_OPEN_MSG = 'channelOpen';
+
 export class AmpViewerHost {
+
```

Comment:
I don't think `AmpViewerHost` is a good name for this. `AmpViewer` itself is the host. This class is currently only responsible for the messaging connection, it looks like.